src/Microsoft.ServiceFabric.Client.Http/ServiceFabricHttpClient.cs

```diff
@@ -574,7 +576,14 @@ private void CreateManagementClients()
             this.CodePackages = new CodePackageClient(this);
             this.ComposeDeployments = new ComposeDeploymentClient(this);
             this.Faults = new FaultsClient(this);
-            this.ImageStore = new ImageStoreClient(this);
+
+            var connectionString = ClientSettings.ImageStoreConnectionString ?? await Cluster.GetImageStoreConnectionString();
```

Comment:
This class should not know about ImageStore conenction string, it should inly be known to IImageStoreClient's concrete implementation,